jobsets/default.nix

```diff
@@ -94,6 +94,14 @@ let
       };
     };
   };
+  mkMantis = mantisBranch: {
+    nixexprpath = "release.nix";
+    nixexprinput = "mantis";
+    description = "Mantis";
+    inputs = {
+      mantis = mkFetchGithub "https://github.com/input-output-hk/mantis-iele-ops.git ${mantisBranch}";
```

Comment:
Best to use a branch of the public [mantis](https://github.com/input-output-hk/mantis) repo.